CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to IRB
+
+Bug reports and pull requests are welcome on GitHub at [https://github.com/ruby/irb](https://github.com/ruby/irb).
+
+## Set up the environment
+
+1. Fork the project to your GitHub account
+2. Clone the fork with `git clone git@github.com:[your_username]/irb.git`
+3. Run `bundle install`
+4. Run `bundle exec rake` to make sure tests pass locally
```

Comment:
Minor: each of these sentences could end in a period. Makes for easier parsing, I think.

Reply:
I've updated but I don't understand about the parsing argument. Do you mind elaborating?

Reply:
Oh, "parse" as in "easier to read for a human reader".